### PR TITLE
add select check to fix edge wave issue

### LINKED ISSUE #14466: The are something wrong in Edge
- [ ] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[Edit on CodeSandbox](https://codesandbox.io/s/rl2j645w6p)

### Steps to reproduce
When a component that relies on Wave re-triggers the animation.

### What is expected?
Compatible with Edge

### What is actually happening?
Can not compatible the environment that does not support removeAttribute

| Environment | Info |
|---|---|
| antd | 3.12.4 |
| React | 16.6.0 |
| System | Win 10 |
| Browser | Edge 17.17134 |

---
https://github.com/ant-design/ant-design/blob/b87261673ababf294f0fbfba680cfd95f7390f72/components/_util/wave.tsx#L44-L45

Need to be compatible with `removeAttribute`

### PR DESCRIPTION
**Pull request by zombieJ**
fix #14466

First of all, thank you for your contribution! 😄

New feature please send pull request to feature branch, and rest to master branch.
Pull request will be merged after one of collaborators approve.
Please makes sure that these form are filled before submitting your pull request, thank you!

[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Branch merge
- [ ] Other (about what?)

### What's the background?

#14466
https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/12087679/

### What's the effect? (Optional if not new feature)

None

### Changelog description (Optional if not new feature)

> 1. Fix edge wave style issue.
> 2. 修复 edge 下水波纹效果的样式问题。

### Self Check before Merge

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/style/core/motion/other.less [hunk 1]
```diff
@@ -4,12 +4,12 @@
   }
 }
 
-[ant-click-animating],
-[ant-click-animating-without-extra-node] {
+[ant-click-animating='true'],
+[ant-click-animating-without-extra-node='true'] {
```
Comment (afc163) [inline components/style/core/motion/other.less]: ' => "

File: components/_util/wave.tsx [hunk 2]
```diff
@@ -115,7 +115,8 @@ export default class Wave extends React.Component<{ insertExtraNode?: boolean }>
     }
     const { insertExtraNode } = this.props;
     const attributeName = this.getAttributeName();
-    node.removeAttribute(attributeName);
+    node.setAttribute(attributeName, 'false');
+    // node.removeAttribute(attributeName);
```
Comment (afc163) [inline components/_util/wave.tsx]: 不要注释

Comment (afc163) [reply components/_util/wave.tsx]: 或者留个注释，解释为啥不用 removeAttribute 。

File: components/_util/wave.tsx [hunk 3]
```diff
@@ -115,6 +115,7 @@ export default class Wave extends React.Component<{ insertExtraNode?: boolean }>
     }
     const { insertExtraNode } = this.props;
     const attributeName = this.getAttributeName();
+    node.setAttribute(attributeName, 'false'); // edge has bug on `removeAttribute` #14466
     node.removeAttribute(attributeName);
```
Comment (afc163) [inline components/_util/wave.tsx]: 这个是不是该删了。

Comment (zombieJ) [reply components/_util/wave.tsx]: 我觉得 `removeAttribute` 更干净，在 edge 无效下通过 `setAttribute` 做 workaround。可以执行 `removeAttribute` 的时候，就清理掉。

Comment (afc163) [reply components/_util/wave.tsx]: 但是这样还是会报 `Object doesn't support property or method 'removeAttribute'` 的错吧。

Comment (zombieJ) [reply components/_util/wave.tsx]: 有道理，直接删了。

File: components/style/core/motion/other.less [hunk 4]
```diff
@@ -4,12 +4,12 @@
   }
 }
 
-[ant-click-animating],
-[ant-click-animating-without-extra-node] {
+[ant-click-animating='true'],
```
[the remark this answers is not in the document]
Comment (zombieJ) [reply components/style/core/motion/other.less]: prettier 自己改的……

**Pull request merged**